AgentFramework/designer.py

```diff
@@ -0,0 +1,111 @@
+import json
```

Comment:
please add information on where did you copy the code from. This is important from IP perspective.

Reply:
add where you copied from in each file.